src/__tests__/no-act.js

```diff
@@ -2,7 +2,7 @@ let act, asyncAct
 
 beforeEach(() => {
   jest.resetModules()
-  act = require('..').act
+  act = require('../pure').act
```

Comment:
`..` would include automatic cleanup i.e. setting up `beforeEach` and `afterEach` hooks. This throws now in jest.